helper/hex/hex.go

```diff
@@ -52,6 +52,17 @@ func EncodeUint64(i uint64) string {
 	return string(strconv.AppendUint(enc, i, 16))
 }
 
+// DecodeUint64 decodes a hex string with 0x prefix to uint64
+func DecodeUint64(hexStr string) uint64 {
+	// remove 0x suffix if found in the input string
+	cleaned := strings.Replace(hexStr, "0x", "", -1)
```

Comment:
`TrimPrefix` is better?

Reply:
Do you mean TrimPrefix is better than cleaned? 
Maybe slicedHex or formattedHex, trimmedHex? What do you think

Reply:
No, I meant strings package had `TrimPrefix` function

Reply:
fied 36cd3c09d6ee00272a1ce9454d16bb685e06ce8f

Reply:
Thanks agree 204fec563dfe954c6af8e159b5f50495fd4631aa